overwatch/processing/trending/manager.py

```diff
@@ -0,0 +1,104 @@
+import logging
+import os
+from collections import defaultdict
+
+import ROOT
+from BTrees.OOBTree import BTree
+from persistent import Persistent
+
+import overwatch.processing.pluginManager as pluginManager
+import overwatch.processing.trending.constants as CON
+
+logger = logging.getLogger(__name__)
+
+try:
+    from typing import *  # noqa
+    from persistent.mapping import PersistentMapping  # noqa
+    from overwatch.processing.processingClasses import histogramContainer  # noqa
+    from overwatch.processing.trending.info import TrendingInfo  # noqa
+    from overwatch.processing.trending.objects.object import TrendingObject  # noqa
```

Comment:
You are right, it may be confusing, because I hadn't written any commnet about it.
It shoud be:
`try:
    from typing import *  # noqa
except ImportError:  
    pass  
else:  
    from persistent.mapping import PersistentMapping  # noqa  
    ...`

Reply:
Okay, I see now - thanks. With this context, your approach makes sense to me. For some additional clarity, I would propose adding a comment such as `# Imports in this block below here are used solely for typing information` right after `from typing import *` whenever it is used so the purpose of those other imports is clear.

Reply:
Added additional comments for this case in 17fd0601293b010f5ef957f43e2d573b31074451